### ISSUE TITLE
尊敬的作者您好，我想请问一下，我的超参数设置是否有问题？我的预测结果与实际值相比相差太多，您能给出建议吗？

### ISSUE THREAD

**Issue by liuchangml** [zhouhaoyi/Informer2020#376]
Namespace(activation='gelu', attn='prob', batch_size=32, c_out=7, checkpoints='./checkpoints/', cols=None, d_ff=2048, d_layers=1, d_model=512, data='ETTh1', data_path='ETTh1.csv', dec_in=7, des="'Exp'", detail_freq='h', devices='0,1,2,3', distil=True, do_predict=False, dropout=0.05, e_layers=2, embed='timeF', enc_in=7, factor=5, features='M', freq='h', gpu=0, inverse=True, itr=1, label_len=96, learning_rate=0.0001, loss='mse', lradj='type1', mix=True, model='informer', n_heads=8, num_workers=0, output_attention=False, padding=0, patience=3, pred_len=336, root_path='./data_csv/', s_layers=[3, 2, 1], seq_len=96, target='OT', train_epochs=6, use_amp=False, use_gpu=True, use_multi_gpu=False)
![image](https://user-images.githubusercontent.com/106809666/194573149-67480d7b-bce9-4c37-b414-d0af591db731.png)
![image](https://user-images.githubusercontent.com/106809666/194573203-aa755f2d-d734-4a76-ac19-debfa2ac733b.png)
![image](https://user-images.githubusercontent.com/106809666/194573331-570dd2bf-43ef-4d56-a66f-6b75bf371307.png)

**Comment by zhouhaoyi** [comment]
可以把部分数据share一下吗？我们可以具体分析一下，最好同时有Dataloder和PE的设置。

**Comment by liuchangml** [comment]
您可以把原程序的main_informer.py和exp_informer.py替换。我的其他很多结果图效果也不太好
 [informer_figure.zip](https://github.com/zhouhaoyi/Informer2020/files/9744624/informer_figure.zip)
@zhouhaoyi

**Comment by liuchangml** [comment]
万分感谢您的回复

**Comment by YUAN0620** [comment]
答主你好，冒昧打扰，我想问一下，如何能得到您所绘制的这个图，我按照原作者的方法，绘制出来的图的纵坐标是在-1到1之间，感觉并不是真实值。万分感谢。

**Comment by strelitziaRT** [comment]
> 答主你好，冒昧打扰，我想问一下，如何能得到您所绘制的这个图，我按照原作者的方法，绘制出来的图的纵坐标是在-1到1之间，感觉并不是真实值。万分感谢。

inverse参数提供反归一化

**Comment by YUAN0620** [comment]
> > 回答主你好，冒昧打错，我想问一下，如何能得到你所绘的这个图，我按原作者的方法，绘出的图的坐标是在-1真到1之间，感觉是视觉值。万分感谢。
> 
> inverse参数提供反归一化

你好，感谢回复！已解决。冒昧问一下，为什么跑出来的预测结果效果很差呢？数据是5min一个步长，用288，144预测72个步长，如下图所示
![4863cd6f2245139290c2d9e0fe2b7cb](https://user-images.githubusercontent.com/83199494/223891616-eb6e47e9-3258-48bc-a2c2-256440d7c104.png)

**Comment by Chao2MAX** [comment]
> > > 回答主你好，冒昧打错，我想问一下，如何能得到你所绘的这个图，我按原作者的方法，绘出的图的坐标是在-1真到1之间，感觉是视觉值。万分感谢。
> > 
> > 
> > inverse参数提供反归一化
> 
> 你好，感谢回复！已解决。冒昧问一下，为什么跑出来的预测结果效果很差呢？数据是5min一个步长，用288，144预测72个步长，如下图所示 ![4863cd6f2245139290c2d9e0fe2b7cb](https://user-images.githubusercontent.com/83199494/223891616-eb6e47e9-3258-48bc-a2c2-256440d7c104.png)

你好，我也遇到了输出数据没有反归一化的情况，请问您是怎么处理的呀？可以教教我吗？

**Comment by elseboy** [comment]
> 
你好，你5min一个步长是怎么设置args.freq的呢？是args.freq = '5t'是吗？

**Comment by elseboy** [comment]
> > > 回答主你好，冒昧打错，我想问一下，如何能得到你所绘的这个图，我按原作者的方法，绘出的图的坐标是在-1真到1之间，感觉是视觉值。万分感谢。
> > 
> > 
> > inverse参数提供反归一化
> 
> 你好，感谢回复！已解决。冒昧问一下，为什么跑出来的预测结果效果很差呢？数据是5min一个步长，用288，144预测72个步长，如下图所示 ![4863cd6f2245139290c2d9e0fe2b7cb](https://user-images.githubusercontent.com/83199494/223891616-eb6e47e9-3258-48bc-a2c2-256440d7c104.png)


你好，你5min一个步长是怎么设置args.freq的呢？是args.freq = '5t'是吗？我看后面又有个5min, 设置会报错KeyError, 只能5t这样子去设置，这样对吗？

**Comment by nixcty** [comment]
答主您好，我在更换自己的数据集的时候模型没有进行predicate操作，但是并没有报错，这是为什么呢？

**Comment by VV1314** [comment]
> > > 回答主你好，冒昧打错，我想问一下，如何能得到你所绘的这个图，我按原作者的方法，绘出的图的坐标是在-1真到1之间，感觉是视觉值。万分感谢。
> > 
> > 
> > inverse参数提供反归一化
> 
> 你好，感谢回复！已解决。冒昧问一下，为什么跑出来的预测结果效果很差呢？数据是5min一个步长，用288，144预测72个步长，如下图所示 ![4863cd6f2245139290c2d9e0fe2b7cb](https://user-images.githubusercontent.com/83199494/223891616-eb6e47e9-3258-48bc-a2c2-256440d7c104.png)

请问一下有绘制图的py文件吗？跑出来得到的不是一个文件吗？
